client/dashboard/sites/settings-wp-version/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { DataForm } from '@automattic/dataviews';
+import { useQuery, useMutation } from '@tanstack/react-query';
+import { notFound } from '@tanstack/react-router';
+import {
+	Card,
+	CardBody,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	Button,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { useState } from 'react';
+import {
+	siteQuery,
+	siteWordPressVersionQuery,
+	siteWordPressVersionMutation,
+} from '../../app/queries';
+import PageLayout from '../../components/page-layout';
+import { getWordPressVersionTagName } from '../../utils/wp-version';
+import SettingsPageHeader from '../settings-page-header';
+import { canUpdateWordPressVersion } from './utils';
+import type { Field, SimpleFormField } from '@automattic/dataviews';
+
+const fields: Field< { version: string } >[] = [
+	{
+		id: 'version',
+		Edit: 'select',
+		elements: [
+			{ value: 'latest', label: getWordPressVersionTagName( 'latest' ) },
+			{ value: 'beta', label: getWordPressVersionTagName( 'beta' ) },
```

Comment:
What is `beta` and why users would want to do that, even if they understand and indeed is a beta WP version before a release?